webapp/maps_heatmaps/package-lock.json

```diff
@@ -16185,4 +16185,4 @@
       "dev": true
```

Comment:
file for other PR?

Reply:
when we deployed the demo using the pipelines this was added to the pr